core/src/main/java/org/mule/runtime/core/processor/strategy/MultiReactorProcessingStrategyFactory.java

```diff
@@ -41,8 +23,9 @@
 public class MultiReactorProcessingStrategyFactory extends ReactorProcessingStrategyFactory {
 
   @Override
-  public ProcessingStrategy create(MuleContext muleContext) {
-    return new ReactorProcessingStrategy(() -> muleContext.getSchedulerService().cpuLightScheduler(),
+  public ProcessingStrategy create(MuleContext muleContext, String schedulersNamePrefix) {
+    return new ReactorProcessingStrategy(() -> muleContext.getSchedulerService()
+        .cpuLightScheduler(config().withMaxConcurrentTasks(1).withName(schedulersNamePrefix + ".event-loop")),
```

Comment:
Merge issue, this implementation should just use cpuLight, no concurrent lask limit.